CRM/Twingle/Page/Profiles.php

```diff
@@ -20,8 +20,11 @@ class CRM_Twingle_Page_Profiles extends CRM_Core_Page {
   public function run() {
     CRM_Utils_System::setTitle(E::ts("Twingle API Profiles"));
     $profiles = [];
-    foreach (CRM_Twingle_Profile::getProfiles() as $profile_name => $profile) {
-      $profiles[$profile_name]['name'] = $profile_name;
+    foreach (CRM_Twingle_Profile::getProfiles() as $profile_id => $profile) {
+      $profiles[$profile_id]['id'] = $profile_id;
+      $profiles[$profile_id]['name'] = $profile->getName();
+      $profiles[$profile_id]['is_default'] = $profile->is_default();
+      $profiles[$profile_id]['selectors'] = $profile->getProjectIds();
       foreach (CRM_Twingle_Profile::allowedAttributes() as $attribute) {
         $profiles[$profile_name][$attribute] = $profile->getAttribute($attribute);
```

Comment:
I think this has been changed in the other PR, maybe let's rebase again - `$profiles` is now indexed with profile IDs, so this needs adjustment.

Reply:
Yes, this change depends on the changes suggested in #73. But what do I need to adjust here? It is already using the `$profile_id` as index.

Reply:
See https://github.com/systopia/de.systopia.twingle/pull/74#discussion_r1296828721 - I'm talking about line 29 (26) which I think needs adjustment.

Reply:
Done! ab5b0b3929b4cd32444bc44bcb0240e376e15f3c